documentation/contributing/future_projects/index.markdown

```diff
@@ -13,10 +13,28 @@ title: Code Sprints
 
 This page lists potential projects that would greatly benefit the MoveIt project. This is by no means an exhaustive list, but is meant to serve as a discussion starting point for code sprints and other efforts. Please contact [PickNik Consulting](https://picknik.ai/connect/) for further information or sponsorship opportunities.
 
-## TrajOpt Integration
+## Creation of a Standard Benchmark Suite for Evaluating MoveIt Performance
+
+The idea is to identify a number of standard tasks at varying levels of difficulty that can be achieved with several robots for which a MoveIt configuration is available. Tasks can range from simply reaching for a particular goal position in a relatively uncluttered space to picking and placing items from cluttered shelves.
+
+Once a standard benchmark has been established, subsequent work can focus on running the benchmark suite as part of a continuous integration suite. Additional work can be done on tuning hyperparameter settings that will lead to better overall performance.
+
+## MoveIt2 Tutorials
```

Comment:
Press kit clarifies you need space before "2"
https://moveit.ros.org/about/press_kit/

Also inconsitent below